leveldb/db.go

```diff
@@ -51,6 +51,7 @@ type DB struct {
 
 	// Stats.
 	aliveSnaps, aliveIters int32
+	writeDelayStat         *wDelayStat
```

Comment:
I think we can just unpack `wDelayStat` struct into DB struct. Also I think using `sync/atomic` is enough (yes, inconsistency between counter and timer might happen but I think that's not a big issue in this case), you can cast `time.Duration` to `int64`.